tests/integration/components/frost-fixed-table-test.js

```diff
@@ -423,10 +423,13 @@ describe(test.label, function () {
     })
 
     it('first column has selection checkboxes', function () {
-      expect($hook('myTable-header-left-selectionCell')).to.have.length(1)
       expect($hook('myTable-left-selectionCell')).to.have.length(heroes.length)
     })
 
+    it('header should have clear selection cell', function () {
```

Comment:
I understand you're just following the conventions of this file but I just wanted to point out it doesn't match the conventions established here https://github.com/ciena-frost/ember-frost-test#it-blocks